packages/aws-cdk/test/integ/run-wrappers/repo/npm

```diff
@@ -33,3 +29,5 @@ elif [[ "$command" == "install" || "$command" == "i" ]]; then
 
   exec $ORIGINAL_NPM $npmargs
 fi
+
+exec $ORIGINAL_NPM "$@"
```

Comment:
This omission was actually a latent bug that was introduced a while back. Basically any `npm` command apart from install would result in a no-op when running against repo.